code/drasil-website/lib/Drasil/Website/CaseStudy.hs

```diff
@@ -59,7 +59,7 @@ data CaseStudy = CS {
 -- so we take the naming scheme from there.
 mkCaseStudy :: Example -> [CaseStudy]
 mkCaseStudy E{choicesE = []} = []
-mkCaseStudy E{sysInfoE = si@SI{_sys = sys}, choicesE = [x]} = [CS{sysInfoCS = si, progName = S $ programName sys, choicesCS = x}]
+mkCaseStudy E{sysInfoE = si@SI{_sys = sys}, choicesE = [x]} = [CS{sysInfoCS = si, progName = S $ abrv sys, choicesCS = x}]
```

Comment:
This was one of the main issues with the website display names. 

Previously, `programName` replaced spaces with underscores. Now it removes them altogether. Hence, keeping `programName` (as done in @3550) here wouldn't work. So, since we're already displaying the human-friendly abbreviation for the case study, we should be free to include our spaces, so we revert to using `abrv`.

Reply:
I reverted this one to keep continuity with the other names in the case studies table containing the final case studies project folder name, but the blurb of discussion is still good.